_posts/2016-07-14-polyrhythms.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "It's Like Patting Your Head and Rubbing Your Tummy: Polyrhythms"
+author: "Hayley MacLeod"
+categories: music
+header: /assets/img/2016-07-15/hypnosis.jpg
+---
+
+So you want to get better at rhythms, eh? Gain a little independence between your hands? Polyrhythms are a great way to start. A polyrhythm is two different rhythmic patterns played simultaneously against one another.
+
+With your attention on the individual beats within the two rhythms, you become entranced by the resulting pattern. Polyrhythms are known for their hypnotic character. It may be why we enjoy listening to them as much as we want to get better at playing them.
+
+![](/assets/img/2016-07-15/cymbal.jpg){:width="60%"}
+
+To begin, start with the most basic of polyrhythms, 2 against 3.
+
+## 2:3
+
+The most accurate way to practice any polyrhythm is with a [metronome](https://pitchplay.io). Look at, listen to, and then try playing the polyrhythm yourself.
```

Comment:
Seems kind of jarring to link the main pitchplay site here. Linking back to the metronome blog, or adding a note that pitchplay has a metronome in parenthesis would better satiate reader expectation.